### ISSUE TITLE
Is is possible to disable 'Edit' button in grid view?

### ISSUE THREAD

**Issue by nick13jaremek** [allcount/allcountjs#117]
When in grid view, we are showing only a subset of the fields for a given Entity type. Then we click on the 'Edit' button and attempt to save one Entity row. We cannot because some of the required fields are not visible in the grid view. 

Of course, when editing the Entity item in the complete view (having access to the fields) no issue appears.

The most straightforward solution would be to remove the 'Edit' button from the grid view. Is this possible?

**Comment by paveltiunov** [comment]
@nick13jaremek yep. This is the case where additional effort is required. The most simple way to remove this button is to override
```
mixin startGridEditingButton()
```

**Comment by nick13jaremek** [comment]
@paveltiunov Thank you for the response! However, I am still stuck with no luck :(

I tried the following:

1. Adding a custom viewPathProvider in my initiatlization file:

```
// Custom views
injection.bindMultiple('viewPaths', ['myViewPathProvider']);
injection.bindFactory('myViewPathProvider', function () {
  return [path.join(__dirname, 'views')];
});
```

2. Created a 'views' folder in the directory of the initiatlization file, matching the returned path for `myViewPathProvider`.

3. Created a `main.jade` file in the `views` directory containing the following content:
```

extends main
include mixins
mixin startGridEditingButton()
```

However, I have not seen the edit button disappear in the grid views. Perhaps I am missing something? A configuration parameter perhaps?

I also tried renaming the `main.jade` file to `mixins.jade` and `entity.jade` (replacing `extends main` with `extends mixins` and `extends entity` respectively) but haven't achieved success.

Any help with this would be very welcome.
